### ISSUE TITLE
Question - Left Join

### ISSUE THREAD

**Issue by fleca** [eveningkid/denodb#103]
Hi, I don't believe this is an issue, but is there a way to perform a left join between 2 Models?

**Comment by stillalivx** [comment]
Hi! I just implement the left join statement to use in SQL database. 

Here is my fork. 
https://github.com/stillalivx/denodb

I you want to use it, is the same way to make a normal join. Example: 
``` javascript
await Business
  .select(Owner.field('name', 'owner'), Business.field('name'))
  .leftJoin(Owner, Owner.field('id'), Business.field('owner_id'))
  .get();
```

**Comment by eveningkid** [comment]
Hey,

Thanks to @stillalivx, left joins are now available in 1.0.8.

I just don't have enough free time to spend on this project at the moment, so thank you for solving this issue.

I wish there would be more people like you to help on such massive projects that could help a bunch of us.

Have a great day
